src/main/java/seedu/address/model/event/Event.java

```diff
@@ -74,6 +74,18 @@ public EventPeriod getEventPeriod() {
         return this.eventPeriod;
     }
 
+    /**
+     * Checks if this event overlaps with another event period
+     *
+     * @param period The other event period.
+     * @return True if there is an overlap, false otherwise.
+     */
+    public boolean isPeriodConflicting(EventPeriod period) {
+        requireNonNull(period);
+
+        return eventPeriod.isOverlapping(period);
+    }
+
```

Comment:
This method seems very similar to the isConflicting method.

Reply:
The isConflicting method has been changed to use this method. Both are required for different uses.

Reply:
Understood that the implementation has to check if an event has conflicts with a time period without having to create a dummy event, therefore necessitating the creation of this method.
Thanks for the clarification!